api/app/controllers/spree/api/v2/storefront/cart_controller.rb

```diff
@@ -24,12 +24,26 @@ def add_item
             end
           end
 
+          def remove_item
+            if spree_current_order.nil?
+              render json: { error: "Order doesn't exist" }, status: 404
+            else
+              line_item = spree_current_order.line_items.find(params[:line_item_id])
```

Comment:
What do you think about moving it to a separate method so that it's easy to override for other developers?
Currently they'll have to override the entire action.